<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>490</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <bold>false</bold>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Image Editor</string>
  </property>
  <property name="windowIcon">
   <iconset theme="applications-graphics"/>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>523</x>
      <y>10</y>
      <width>20</width>
      <height>361</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>711</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="ToolsLine">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>150</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="ToolsBox">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>170</y>
      <width>181</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>171</width>
       <height>151</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="FiltersButton">
        <property name="text">
         <string>Filters</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ColorsButton">
        <property name="text">
         <string>Color Adjustment</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="CropButton">
        <property name="text">
         <string>Crop</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ResizeButton">
        <property name="text">
         <string>Resize</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>181</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Controls</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>171</width>
       <height>104</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="SelectButton">
        <property name="text">
         <string>Select Image</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="DownloadButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Download</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="EditButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Edit</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="ImageLable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>511</width>
      <height>361</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="FiltersBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>711</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Filters</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QDial" name="BlurDial">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Blur :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>40</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Gray Scale :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Sharpen :</string>
     </property>
    </widget>
    <widget class="QDial" name="SharpenDial">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>20</width>
       <height>41</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_5">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>30</y>
       <width>20</width>
       <height>41</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="blurLabel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0%</string>
     </property>
    </widget>
    <widget class="QLabel" name="sharpenLabel">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0%</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="GrayCheckBox">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>40</y>
       <width>76</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ColorsBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>711</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Color Adjustment</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QDial" name="satDial">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>40</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Saturation :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>40</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Contrast</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Brightness :</string>
     </property>
    </widget>
    <widget class="QDial" name="brightDial">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>20</width>
       <height>41</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_6">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>30</y>
       <width>20</width>
       <height>41</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="satLabel">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0%</string>
     </property>
    </widget>
    <widget class="QLabel" name="brightLabel">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>40</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0%</string>
     </property>
    </widget>
    <widget class="QLabel" name="contLabel">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>40</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0%</string>
     </property>
    </widget>
    <widget class="QDial" name="contDial">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>10</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="CropBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>711</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Crop</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="cropCropButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Crop</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelCropButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ResizeBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>711</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Resize</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>20</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="resizeResizeButton">
        <property name="text">
         <string>Resize</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLineEdit" name="HeightQL">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>22</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
    <widget class="QLineEdit" name="wdithQL">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>50</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
    <widget class="QLabel" name="imageHL">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="imageWL">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="widthL">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>50</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Width :</string>
     </property>
    </widget>
    <widget class="QLabel" name="heightL">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>20</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Height :</string>
     </property>
    </widget>
    <widget class="QLabel" name="imageHL_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="imageWL_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>50</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="cropFrame">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>730</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
